code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -41,8 +41,6 @@
 	if(stage > 1 && !active && clown_check(user))
 		user << SPAN_WARNING("You prime \the [name]!")
 
-		msg_admin_attack("[user.name] ([user.ckey]) primed \a [src]. (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)")
-
```

Comment:
That's kind of unrelated change.

Reply:
i love unrelated changes.
This message is not needed since missing igniter. It wont go off so there is no need for it.

Reply:
Happy for you, but all changes should be related to PR topic.
Or at least mentoinded in a description.